src/main/java/frc/robot/constants/fields/TheoreticalField.java

```diff
@@ -0,0 +1,17 @@
+package frc.robot.constants.fields;
+
+import edu.wpi.first.wpilibj.geometry.Translation2d;
+import frc.robot.constants.FieldConstants;
+
+/**
+ * The field way-points in theory. relative to the (0,0) point in PathWeaver.
+ * all measurement are measured in meters.
+ */
+public class TheoreticalField extends FieldConstants {
```

Comment:
I think that if it's a TheroticalField it should use measurements from the manual or the official CAD of the field